### ISSUE TITLE
Tube Feeding

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5890]
**Preferred term label:** Tube Feeding

**Synonyms** Enteral tube feeding

**Definition (free text, please give PubMed ID)** Feeding problem necessitating food and nutrient delivery via a tube., PMID: 28044031; PMID: 25874832

**Parent term (use hpo.jax.org/app)** Feeding difficulties HP:0011968

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
Tube feeding could be the parent term for: Nasogastric tube feeding HP:0040288

**Comment by JillMiller** [comment]
**Preferred term label:** Nasojejunal tube feeding

**Synonyms** NJ tube feeding

**Definition (free text, please give PubMed ID)**  The condition of the inability to eat normally necessitating the placement of a thin, soft tube into the nose that goes through the stomach and ends up in the jejunum. This tube is then used to carry nutrients to be absorbed by the small intestine., PMID: 30011238

**Parent term (use hpo.jax.org/app)** Tube Feeding

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Gastrostomy tube feeding

**Synonyms** G-tube feeding 

**Definition (free text, please give PubMed ID)** Feeding problem necessitating gastrostomy tube feeding., PMID: 27184363

**Parent term (use hpo.jax.org/app)** Tube Feeding

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Gastrojejunal tube feeding

**Synonyms** JG-tube feeding

**Definition (free text, please give PubMed ID)** Feeding problem necessitating gastrojejunal tube feeding., PMID: 26232359; PMID: 26227672

**Parent term (use hpo.jax.org/app)** Tube Feeding

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP_0033454
